### PR TITLE
Add pre-release notes

### PR DESCRIPTION
**Pull request by Senyoret1**
Changes:
- This pr adds to `README.md` notes about the tests that should be done before launching a new release.

### CODE REVIEW

**Comment by Senyoret1** [comment]
https://github.com/skycoin/skycoin-web/pull/501 and https://github.com/skycoin/skycoin-web/pull/502 must be merged first

File: README.md [hunk 1]
```diff
@@ -74,3 +74,13 @@ with enough entropy, that library uses the [randombytes library](https://www.npm
 cryptographically strong random values. The use of the [bip39 library](https://www.npmjs.com/package/bip39) is limited to
 the creation of mnemonic phrases, the code that creates the binary seeds is in
 [skycoin-lite](https://github.com/skycoin/skycoin-lite).
+
+## Pre-release testing
+
+Performs these actions before releasing:
```
Comment (mahansky) [inline README.md]:
```suggestion
Perform these actions before releasing:
```

File: README.md [hunk 2]
```diff
@@ -74,3 +74,13 @@ with enough entropy, that library uses the [randombytes library](https://www.npm
...
+
+- Test if all options are working correctly when opening the wallet on the test server (using `npm start`). Load a wallet with balance and try to send coins, to verify if the functions for loading wallets and creating of transactions work correctly.
+
+- Test if all options are working correctly when opening the wallet from the local file system (the compilation is done by executing `make build-for-local-fs`). In the local version it is important to check all the pages and make sure that all the images load correctly. For this it is useful to go through all pages while the development console is open.
+
+- Create/load a wallet in a production build (it can be the one created by `make build-for-local-fs`) and then refresh the browser. The wallet should return to the wizard, which means that the data is not kept permanently.
```
Comment (mahansky) [inline README.md]: This step has an e2e test so it doesn't have to be here, what do you think?

Comment (Senyoret1) [reply README.md]: You are right. When I wrote the notes I had not yet uploaded the commit with the e2e test and forgot to update the text.

**Pull request merged**